src/main/kotlin/org/jitsi/rtp/rtcp/rtcpfb/payload_specific_fb/RtcpFbRembPacket.kt

```diff
@@ -87,15 +84,15 @@ class RtcpFbRembPacket(
         const val NUM_SSRC_OFF = REMB_OFF + REMB_LEN
         const val NUM_SSRC_LEN = 1
         const val BR_OFF = NUM_SSRC_OFF + NUM_SSRC_LEN
-        const val BR_LEN = 4
-        const val SSRCS_OFF = NUM_SSRC_OFF + BR_LEN
+        const val BR_LEN = 3
+        const val SSRCS_OFF = BR_OFF + BR_LEN
 
         fun getBrExp(buf: ByteArray, baseOffset: Int): Int =
             buf.getBitsAsInt(baseOffset + BR_OFF, 0, 6)
         fun getBrMantissa(buf: ByteArray, baseOffset: Int): Int =
-            (buf.getBitsAsInt(baseOffset + BR_OFF, 6, 2) shl 2) + buf.getShortAsInt(baseOffset + BR_OFF + 1)
+                (buf.getBitsAsInt(baseOffset + BR_OFF, 6, 2) shl 16) + buf.getShortAsInt(baseOffset + BR_OFF + 1)
         fun getBitrate(buf: ByteArray, baseOffset: Int): Long =
-            (getBrMantissa(buf, baseOffset) * Math.pow(2.0, getBrExp(buf, baseOffset).toDouble())).toLong()
+            (getBrMantissa(buf, baseOffset) shl getBrExp(buf, baseOffset)).toLong()
```

Comment:
The old code handled the case where the bitrate overflows a `Long`, whereas this doesn't.  (REMB bitrates can be up to 2^18-1 * 2^63 ≈ 2^81.)

Clamping our interpretation of the bitrate at MAX_LONG is fine for these very large values, but I think this code shifts the high-order bits off the mantissa.

Reply:
There's probably something wrong with the old code as well since testing at the limits with Long.MAX_VALUE fails

Reply:
I partially reverted my change so we should be good for now, but we may want to go back to it later because I'm not confident that this code works correctly in general nor that it is the most elegant (especially the part that finds the exponent and the mantissa).

Reply:
I'm confused...so do we have a known issue with large numbers here now or not?  Can we just add a unit test to check?

Reply:
> I'm confused...so do we have a known issue with large numbers here now or not?

yeah there was an issue

> Can we just add a unit test to check?

👍

Reply:
Is there a test for the -1 case? I didn't see it but maybe I missed it.

Reply:
you didn't miss it, i just added one